src/csharp/Microsoft.Spark/Sql/DataFrame.cs

```diff
@@ -805,12 +805,29 @@ public DataFrame RepartitionByRange(int numPartitions, params Column[] partition
         /// <returns>DataFrame object</returns>
         public DataFrame Persist() => WrapAsDataFrame(_jvmObject.Invoke("persist"));
 
+        /// <summary>
+        /// Persist this `DataFrame` with the given storage level.
+        /// </summary>
+        /// <param name="storageLevel">Storage level to persist the dataframe to.</param>
+        /// <returns>DataFrame object</returns>
+        public DataFrame Persist(StorageLevel storageLevel) => 
+            WrapAsDataFrame(_jvmObject.Invoke(
```

Comment:
nit: No need to break? `WrapAsDataFrame(_jvmObject.Invoke("persist", storageLevel));`